vagga.yaml

```diff
@@ -24,3 +46,26 @@ commands:
     description: Shortcut to manage.py
     container: django
     run: [python3, manage.py]
+  run-postgres: !Supervise
+    description: Start the django development server using Postgres database
+    children:
+      app: !Command
+        container: django
+        environ:
```

Comment:
Nit: you could add this right into container description, since this is not the only command needing this var